final_working_test.py

```
@@ -0,0 +1,131 @@
+    try:
+        response = requests.post(login_url, json=login_payload, timeout=20)
+        if response.status_code != 200:
+            print(f"Login failed: {response.status_code}")
+            return False
+
+        token = response.json().get("access_token")
+        user_id = response.json().get("user_id")
+        headers = {"Authorization": f"Bearer {token}"}
+        print(f"[OK] Authenticated: {user_id}")
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)